kahuna/public/js/leases/leases.css

```diff
@@ -0,0 +1,106 @@
+.lease__form select,
+.lease__form input,
+.lease__form .lease__date {
+  box-sizing: border-box;
+  width: 100%;
+  margin-bottom: 10px;
+}
+
+.leases__wrapper gr-icon:hover {
+  cursor: pointer;
+}
+
+.lease__item {
+  width: 100%;
+  margin: 3px;
+  background-color: #565656;
+}
+
+.lease__item p {
+  margin: 2px;
+}
+
+.lease__date {
+  position: relative;
+}
+
+.add-lease__header {
+  display: block;
+}
+
+.lease {
+  margin: 4px;
+  padding: 3px;
+  border-top-left-radius: 2px;
+  border-bottom-left-radius: 2px;
+}
+
+.lease:hover {
+  color:white;
+}
+
+gr-leases .metadata-line__key {
+  margin-bottom: 10px
+}
+
+gr-leases gr-confirm-delete {
+  float: right;
+}
+
+.active.allowed {
+  background-color: lightgreen;
+}
+
+.active.denied {
+  background-color: red;
+}
+
+gr-leases .gu-date-range__overlay {
+  position: absolute;
+  z-index: 1;
+  right: 0px;
+}
+
+gr-leases .gu-date-range__display {
+  width: 100%;
+}
+
+.image-details--full-image,
+.image-info__wrap {
+  overflow: visible;
+}
+
+.lease__form__buttons {
+  text-align: right;
+  margin-top: 8px;
+  margin-bottom: 20px;
+}
+
+.lease__delete {
+  float: right;
+}
+
+.gr-confirm-delete:hover :not(gr-icon--small){
+  background-color: inherit;
+}
+
+.lease__access,
+.lease__current {
+  box-sizing: border-box;
+  width: 100%;
+  float: left;
+}
+
+.lease__access {
+  padding: 3px;
+  margin-bottom: 7px;
+}
+
+.lease__current {
+  padding: 2px;
+
+}
+
+.current {
```

Comment:
Is this a modifier for an existing block?/ Should be `block--current` if so to group it.